app/src/main/java/com/devshish/internship/domain/util/Constants.kt

```diff
@@ -0,0 +1,10 @@
+package com.devshish.internship.domain.util
+
+object Constants {
+
+    const val BASE_URL = "https://api.genius.com"
```

Comment:
1. Try to move Base Url to build.gradle (app) file 
2. And remove these constants, as I write previously